app/main.py

```diff
@@ -1,3 +1,57 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def read_config_file(file_path: str) -> dict:
+    with open(file_path, "r") as file:
+        data = json.load(file)
+    return data
+
+
+def create_customers(customers_data: list) -> list:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]